xls/modules/zstd/BUILD

```diff
@@ -1428,12 +1428,6 @@ xls_dslx_test(
     ],
 )
 
-xls_dslx_library(
```

Comment:
is that replaced w/ ram demux?

Reply:
with `RamPassthrough` ([code](https://github.com/google/xls/pull/2653/files/0d1fc31ec459b803af5cb041934830a2a9753bdf#diff-cb74ede7463d11191318780b775a62a1139076d590ea087065aaefe167a36c4fR1399))